docs/html/md__compiler.html

```diff
@@ -78,6 +78,7 @@
 <li><code>--strict-json</code> : Require &amp; generate strict JSON (field names are enclosed in quotes, no trailing commas in tables/vectors). By default, no quotes are required/generated, and trailing commas are allowed.</li>
 <li><code>--defaults-json</code> : Output fields whose value is equal to the default value when writing JSON text.</li>
 <li><code>--no-prefix</code> : Don't prefix enum values in generated C++ by their enum type.</li>
+<li><code>--scoped-enums</code> : Use C++11 style scoped and strongly typed enums in generated C++. This also implies <code>--no-prefix</code>.</li>
```

Comment:
Since there was already an option `--no-prefix` to control enum generation I was inclined to use another one specific to enum to make it explicit. How about replacing `--no-prefix` with something like `--gen-enums noprefix|scoped` though that would be a breaking change. If that doesn't go well, I can change the option per your suggestion to `--use-latest-cpp` or `--gen-latest-cpp`.